spec/services/accessorizer_spec.rb

```diff
@@ -94,7 +94,7 @@
 
     context "a quantity accessory" do
       let(:params) do
-        HashWithIndifferentAccess.new(quantity_accessory.id.to_s => { enabled: "true", quantity: "3" })
+        ActionController::Parameters.new(quantity_accessory.id.to_s => { enabled: "true", quantity: "3" })
```

Comment:
Do we need the `to_s` on the ID?

Reply:
Specs fail without it, so I'm thinking yes.